dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -1435,6 +1435,33 @@ public Iterator<Item> findByMetadataField(Context context,
         }
     }
 
+    @Deprecated(forRemoval = true)
+    @Override
+    public Iterator<Item> findByMetadataQuery(Context context, List<List<MetadataField>> listFieldList,
```

Comment:
@jeffmorin : You appear to have accidentally restored a method that was **deleted** in this PR #9258 

I don't see any usages of this `findByMetadataQuery()` in your code.  Could your remove it again along with the corresponding code in ItemDAO/ItemDAOImpl?

Unfortunately, this code is NOT compatible with the latest Hibernate.  It's using tools that were removed in Hibernate v6.  So, we must remove it.

Reply:
I saw the old dspace-rest module was removed, but I had completely forgotten about the methods that I indeed marked as deprecated. I will clean this up and let you know once it's done and committed to my fork.